app/code/Magento/Catalog/Model/Product/Option.php

```diff
@@ -135,14 +148,18 @@ public function __construct(
         \Magento\Framework\Model\ResourceModel\AbstractResource $resource = null,
         \Magento\Framework\Data\Collection\AbstractDb $resourceCollection = null,
         array $data = [],
-        ProductCustomOptionValuesInterfaceFactory $customOptionValuesFactory = null
+        ProductCustomOptionValuesInterfaceFactory $customOptionValuesFactory = null,
+        array $optionGroups = [],
```

Comment:
Actually someone could override this class in own module and not pass these two arguments, in this case - we'll have empty array here. please add default value, similar to getting objects from object manager, but in this case it will be default array

Reply:
Understood. Sorry for such issues, had no experience in core contribution